examples/cpp/Makefile

```diff
@@ -1,17 +1,20 @@
-MOLE = ../../src/cpp
-
-CXX ?= g++
+MOLE = ../../src/mole_C++
+#ARMA = /Users/aneeshsrinivas/Documents/GitHub/armadillo-14.0.3
+#CXX ?= clang++
```

Comment:
Hi @aneeshs1729 , was line 3 commented intentionally? Is that MacOS specific?

Reply:
yes, i did have to change it in order to run my examples.

Reply:
Hi @aneeshs1729 , what is not clear to me is that you had to change it to use the clang++ compiler, but then you commented it? So this line is not an effective change in your Makefile. Please delete it, if not needed. Or leave a comment that it is specific to MacOS like you did for other lines. Thank you